std/stdio.d

```diff
@@ -4607,7 +4607,7 @@ struct lines
     f = File to read lines from.
     terminator = Line separator (`'\n'` by default).
     */
-    this(File f, dchar terminator = '\n')
+    this(File f, dchar terminator = '\n') @safe @nogc pure
```

Comment:
```
std/stdio.d(4610): Error: `pure` constructor `std.stdio.lines.this` cannot call impure destructor `std.stdio.File.~this`
std/stdio.d(4610): Error: `@nogc` constructor `std.stdio.lines.this` cannot call non-@nogc destructor `std.stdio.File.~this`
std/stdio.d(4610): Error: `pure` constructor `std.stdio.lines.this` cannot call impure destructor `std.stdio.lines.~this`
std/stdio.d(4599):        generated `lines.~this` is impure because of the following field's destructors:
std/stdio.d(4601):         - File f
std/stdio.d(475):           impure `File.~this` is declared here
std/stdio.d(4610): Error: `@nogc` constructor `std.stdio.lines.this` cannot call non-@nogc destructor `std.stdio.lines.~this`
std/stdio.d(4599):        generated `lines.~this` is non-@nogc because of the following field's destructors:
std/stdio.d(4601):         - File f
std/stdio.d(475):           non-@nogc `File.~this` is declared here
```

Reply:
ugh Let's see if at least `@safe` works.